model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -325,8 +327,8 @@ def lift_up_through(node: Node, reverse_channels: Node):
             node_input_port_0.get_connection().set_source(reverse_channels.out_port(0))
             src.connect(reverse_channels.in_port(0))
             node.out_port(0).get_connection().set_destination(reverse_channels_out_npde.in_port(0))
-            return True
-        return False
+            return True, [reverse_channels_out_npde]
```

Comment:
@jane-intel , please, double check that the second return value here is correct.